index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express')
+const cors = require('cors')
+
+const contactRouter = require('./routes/contact')
+const articleRouter = require('./routes/article')
+
+const app = express()
+const port = process.env.PORT || 3000
```

Comment:
How does this work? If there is nothing in process.env.PORT does it default to 3000?

Reply:
Yeah. If you set a variable in terminal called `PORT` it'll use that, otherwise 3000.